src/main/java/mate/academy/spring/model/Role.java

```diff
@@ -0,0 +1,40 @@
+package mate.academy.spring.model;
+
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "roles")
+public class Role {
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-spring/security-part-2/jv-spring-security-checklist)
4. Better use Set of roles in the user model, instead of the List (**don’t forget about equals and hashcode**).